gem/lib/earlgrey/configure_earlgrey.rb

```diff
@@ -378,7 +378,12 @@ def copy_swift_files(project, target, swift_version = nil)
       src_swift_name = 'EarlGrey.swift'
       src_swift = File.join(src_root, "Swift-#{swift_version}", src_swift_name)
 
-      raise 'Bundled swift missing' unless File.exist? src_swift
+      unless File.exist? src_swift
+        puts_magenta "Swift version #{swift_version} not found, falling back to 3.0."
```

Comment:
EarlGrey.swift for version #{swift_version} not found. Falling back to version 3.0.